drivers/gles3/rasterizer_storage_gles3.cpp

```diff
@@ -3716,10 +3717,11 @@ void RasterizerStorageGLES3::canvas_light_occluder_set_polylines(RID p_occluder,
 	co->lines = p_lines;
 
 	if (p_lines.size() != co->len) {
-		if (co->index_id)
+		if (co->index_id) {
 			glDeleteBuffers(1, &co->index_id);
-		if (co->vertex_id)
+		} if (co->vertex_id) {
```

Comment:
Small mistake here but it's commented out code anyway, and the code is still valid. Would be fixed by `clang-format` if/when this code is uncommented. So I'll fix it only if I spot another issue that warrants a force push, otherwise we can preserve some CI :)